src/app/content/components/Toolbar/styled.tsx

```diff
@@ -324,6 +324,7 @@ export const InnerText = styled.div`
 
 // tslint:disable-next-line:variable-name
 export const SidebarControl = styled(OpenSidebarControl)`
+  order: -1;
```

Comment:
i haven't used the order property before but it does seem perfect for what we're doing, good find